src/alandipert/kahn.cljd

```diff
@@ -0,0 +1,69 @@
+;; Copyright (c) Alan Dipert. All rights reserved.
+;; The use and distribution terms for this software are covered by the
+;; Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+;; By using this software in any fashion, you are agreeing to be bound by
+;; the terms of this license.
+;; You must not remove this notice, or any other, from this software.
+
+(ns alandipert.kahn
```

Comment:
Should there be a link to the original gist? Not sure

Reply:
I'll add the link to the gist anyway